base/libgit2/blob.jl

```diff
@@ -8,10 +8,34 @@ function Base.length(blob::GitBlob)
     return ccall((:git_blob_rawsize, :libgit2), Int64, (Ptr{Void},), blob.ptr)
 end
 
+"""
+Use a heuristic to guess if a file is binary: searching for NULL bytes and
```

Comment:
Why should we give the sig, when this function probably will never be exported and is "experts-only"? It's not going into `doc/src/stdlib`...